runtime/gc_realtime/Scheduler.cpp

```diff
@@ -20,37 +20,25 @@
  * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
  *******************************************************************************/
 
-#include "j9.h"
-#include "j9cfg.h"
-#include "j9protos.h"
-#include "j9consts.h"
-#include "modronopt.h"
-#include "ModronAssertions.h"
```

Comment:
I would not remove this include, code in this file contains MM Assertions

Reply:
Okay, updated